ToDoList/src/features/toDoSlice.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-console */
+import { createSelector, createSlice } from '@reduxjs/toolkit';
+import { StatusFilters } from './filterSlice';
+import 'regenerator-runtime/runtime';
+
+const initialState = {
+  toDoList: [],
+  status: 'idle',
+};
+
+const toDoSlice = createSlice({
+  name: 'todos',
+  initialState,
+  reducers: {
+    saveToDo: (state, action) => {
+      state.toDoList.push(action.payload);

```

Comment:
может здесь получить lastId через state.toDoList и выполнить логику, чем тянуть этот lastId через action в санку

``
```suggestion
      const lastItemId =  state.toDoList[state.toDoList.length - 1].id
      state.toDoList.push({
         ...action.payload,
         id: lastItemId + 1
      });
     
```